src/haz3lcore/zipper/Selection.re

```diff
@@ -2,8 +2,8 @@ open Util;
 
 [@deriving (show({with_path: false}), sexp, yojson)]
 type buffer =
-  | Unparsed
-  | Parsed;
+  //| Parsed
```

Comment:
what's going on here?

Reply:
This is an artefact of separating the original TyDi implementation from the llm completion stuff. The parsed buffer option is currently meaningless so I just removed it until the LLM stuff is reimplemented